<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnentrar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(164, 0, 0);
border-radius: 10px;
border-style: outset;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uselog">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>            Nome do Usuário</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="senhalog">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>             Senha do Usuário</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>30</y>
     <width>401</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: qlineargradient(spread:pad, x1:0.065, y1:0.727, x2:1, y2:0, stop:0 rgba(234, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(136, 138, 133);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="infor">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>249</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="usecadastrar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(252, 233, 79);
border-radius: 10px;
border-style: outset;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string>Cadastrar Novo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; color:#204a87;&quot;&gt;Seja Bem Vindo!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>png-clipart-delta-computers-rentals-computer-icons-user-icon-entrepreneurship-head-desktop-wallpaper.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>png-transparent-computer-icons-font-awesome-password-user-font-others-typeface-computer-icons-symbol.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>220</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>efeito novo.png</pixmap>
   </property>
  </widget>
  <zorder>pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>usecadastrar</zorder>
  <zorder>btnentrar</zorder>
  <zorder>senhalog</zorder>
  <zorder>uselog</zorder>
  <zorder>infor</zorder>
 </widget>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
